docusaurus.config.js

```diff
@@ -10,15 +10,15 @@ const editUrl = {
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: "🚧 cf-infra-docs 🚧",
+  title: "conda-forge documentation",
```

Comment:
I would like to keep this temporary title to reflect that this is only a prototype so far.